### PR TITLE
Set up tokens for layout/container styles

### LINKED ISSUE #3178: Update/refactor backend container styles
Class names: `.view` and `.layout` in the backend.

These contain fallback styles that would probably be better left unset. Additionally, it's probably better to use `utrecht-document` and fill in the design tokens for that component in our own theme, replacing our styles.

It's probably also recommended to always include the SDK CSS bundle and use the `.openforms-layout` components, reducing CSS duplication/divergence.

The `:root` CSS setting design tokens should also be properly scoped under the `.openforms-theme` (probably).

- [x] Emit run-time design tokens under theme class name instead of `:root`
- [x] Refactor page layout to `utrecht-page` container
- [x] Refactor header to use `utrecht-page-header`
- [x] Refactor footer to use `utrecht-page-footer`
- [x] Refactor content scaffolding to `utrecht-page-content`
    - Remove horizontal padding from `openforms-layout`?
- #3593
- [x] Document token upgrade/mapping paths

### PR DESCRIPTION
**Pull request by sergei-maertens**
Partly closes open-formulieren/open-forms#3178

Required for SDK changes
Required for backend changes

### CODE REVIEW

File: src/community/utrecht/page-content.tokens.json
Comment (sergei-maertens) [inline src/community/utrecht/page-content.tokens.json:1]: This is the page container in the backend.

File: src/community/utrecht/page-footer.tokens.json
Comment (sergei-maertens) [inline src/community/utrecht/page-footer.tokens.json:1]: This is the page footer in the backend.

File: src/community/utrecht/page-header.tokens.json
Comment (sergei-maertens) [inline src/community/utrecht/page-header.tokens.json:1]: This is the page header in the backend.

File: src/community/utrecht/page.tokens.json
Comment (sergei-maertens) [inline src/community/utrecht/page.tokens.json:1]: This is the page container in the backend.

File: src/components/app.tokens.json
Comment (sergei-maertens) [inline src/components/app.tokens.json:1]: SDK app container, containing the useful bit, language switcher and debug div for development.

**Review by SilviaAmAm — APPROVED** [review]

**Pull request merged**